### PR TITLE
Provide minimized stack lines

### PR DESCRIPTION
**Pull request by bmc-msft**
Using the function name only method for minimization results in a
function with two different malloc related bugs to get bucketed
together.

This provides a minimized stack without function address of stack depth.

For entries without source information, such as:
`# 34 0xf19113f in ChromeMain+0x13f (C:\\clusterfuzz\\bot\\builds\\chrome-test-builds_media_win32-release\\revisions\\asan-win32-release-335593\\chrome_child.dll+0x113f)`
users will see this:
`ChromeMain+0x13f`

For entries with source information, such as:
`# 20 0x58bd3bb in v8::internal::Builtin_Impl_HandleApiCall(v8::internal::BuiltinArguments, v8::internal::Isolate*) ../../v8/src/builtins/builtins-api.cc:137:5`
users will see this:
`v8::internal::Builtin_Impl_HandleApiCall(v8::internal::BuiltinArguments, v8::internal::Isolate*) builtins-api.cc:137:5`

### CODE REVIEW

**Comment by ranweiler** [comment]
Will review this after conflicts resolved; please tag me in when ready!

File: src/agent/libclusterfuzz/third-party/update.sh [hunk 1]
```diff
@@ -12,6 +12,7 @@ git clone --depth 1 https://github.com/google/clusterfuzz clusterfuzz-src
 mv clusterfuzz-src/src/python/lib/clusterfuzz/stacktraces/constants.py .
 mkdir -p ../data/stack-traces
 cp clusterfuzz-src/src/python/tests/core/crash_analysis/stack_parsing/stack_analyzer_data/*.txt ../data/stack-traces/
+chmod -x ../data/stack-traces/*.txt
```
Comment (ranweiler) [inline src/agent/libclusterfuzz/third-party/update.sh:15]: What's going on here?

**Review by ranweiler — APPROVED** [review]

File: src/agent/libclusterfuzz/third-party/update.sh (hunk 1 above)
Comment (bmc-msft) [reply src/agent/libclusterfuzz/third-party/update.sh:15]: some portion of the text files in the clusterfuzz repo are executable.  this fixes that.

Comment (ranweiler) [reply src/agent/libclusterfuzz/third-party/update.sh:15]: Oh wow, I misread the `-` as a `+`. That makes _way_ more sense.

**Pull request merged**